LICENSE.md

```diff
@@ -1,7 +1,7 @@
-#Copyright © 2003-2016 LogMeIn, Inc.
+# Copyright © 2003-2016 LogMeIn, Inc.
```

Comment:
Copyright 2017 please.

Reply:
We can't label it 2018 yet, right?

There's still A LOT of 2016 headers ( https://gist.github.com/Palantir555/5e5d2b14c3f62b84fcc3c500b26e7a77 ).
We may as well wait for a week and update them all with a script.